tests/unit/MigrationTest.js

```diff
@@ -117,37 +126,146 @@ describe('Migrations', () => {
                             expect(allReports.report_2.lastActionCreated).toBe('2022-11-16 01:31:54.821');
                         },
                     });
-                });
-        });
+                })
+        ));
 
-        it('Should skip if the report data already has the correct fields', () => {
-            Onyx.set(ONYXKEYS.COLLECTION.REPORT, {
-                report_1: {
+        it('Should skip if the report data already has the correct fields', () => (
+            Onyx.multiSet({
+                [`${ONYXKEYS.COLLECTION.REPORT}1`]: {
                     lastActionCreated: '2022-11-16 01:31:13.702',
                 },
-                report_2: {
+                [`${ONYXKEYS.COLLECTION.REPORT}2`]: {
                     lastActionCreated: '2022-11-16 01:31:54.821',
                 },
             })
                 .then(AddLastActionCreated)
                 .then(() => {
                     expect(LogSpy).toHaveBeenCalledWith('[Migrate Onyx] Skipped migration AddLastActionCreated');
-                });
-        });
+                })
+        ));
 
-        it('Should work even if there is no report data', () => {
+        it('Should work even if there is no report data', () => (
             AddLastActionCreated()
                 .then(() => {
                     expect(LogSpy).toHaveBeenCalledWith('[Migrate Onyx] Skipped migration AddLastActionCreated');
                     const connectionID = Onyx.connect({
                         key: ONYXKEYS.COLLECTION.REPORT,
-                        waitForCollectionCallbacks: true,
+                        waitForCollectionCallback: true,
                         callback: (allReports) => {
                             Onyx.disconnect(connectionID);
                             expect(allReports).toBeEmpty();
                         },
                     });
-                });
-        });
+                })
+        ));
+    });
+
+    describe('KeyReportActionsByReportActionID', () => {
```

Comment:
NAB: should we be adding one test file per migration instead of having them all in one file?

Reply:
Maybe, but I'm just continuing a pattern that's already here. I think we could do a fair amount of test + directory cleanup, but I'm going to skip that for now, since that's not really part of my objective with this PR. Also there's some shared test setup that would be duplicated if I separated this out.